spec/models/allyship_spec.rb

```diff
@@ -49,4 +49,60 @@
 	  new_allies = build(:allyships_accepted, user_id: new_user1.id, ally_id: nil)
 	  expect(new_allies).to have(1).error_on(:ally_id)
 	end
+
+  context 'when destroying' do
```

Comment:
Looks like we should also test that the ally is being removed from viewers lists too.